Initial app loading indicator (#787)
SAP/luigi: docs/luigi-ux-features.md

```

* Define and apply [responsiveNavigation](./general-settings.md) settings to make the left navigation responsive.

### App loading indicator

To show a loading indicator before Luigi Core or your first micro frontend is ready, add a container with the `luigi-app-loading-indicator` attribute to your _index.html_ body or inside your [`luigi-app-root`](#rendering-of-luigi-application-in-the-dom) container.

```html
<div luigi-app-loading-indicator>
  <div class="fd-spinner">
    <div></div>
  </div>
</div>
```

By default, the loading indicator is removed after `Luigi.setConfig({})` has been executed. 

> Alternatively, to keep the loading indicator until the first micro frontend is usable, follow the following steps:

Reviewer: ```suggestion
Alternatively, to keep the loading indicator until the first micro frontend is usable, follow these steps:
```

> 1. Set app loading indicator `hideAutomatically` to `false`

Reviewer: ```suggestion
1. Set the app loading indicator parameter `hideAutomatically` to `false`
```

```javascript
{
  ...
  settings: {
    appLoadingIndicator: {
      hideAutomatically: false
    }
  }
  ...
}
```

> 2. Call [`Luigi.ux().hideAppLoadingIndicator()`](./luigi-core-api.md#hideAppLoadingIndicator) in the Luigi Core once your initial micro frontend has finished loading to remove the loading indicator. You can for example use the [custom messages](./communication.md#custom-messages) feature for the micro frontend using Luigi Client to communicate the Core when this function should be executed.

Reviewer: ```suggestion
2. Call [`Luigi.ux().hideAppLoadingIndicator()`](./luigi-core-api.md#hideAppLoadingIndicator) in the Luigi Core once your initial micro frontend has finished loading to remove the loading indicator. You can, for example, use the [custom messages](./communication.md#custom-messages) feature to allow the Luigi Client micro frontend to communicate with the Core when this function should be executed.
```

Author: done
